cli/command/image/remove_test.go

```diff
@@ -68,15 +77,6 @@ func TestNewRemoveCommandSuccess(t *testing.T) {
 			},
 		},
 		{
-			name: "Image Deleted with force option",
```

Comment:
Thank you for the review. I added a test case. 
Would it be a good idea for the docker/docker/client package to export the notFound* errors to make testing easier? I think in addition to the new test, _stack/deploy_composefile_test.go_ also implements a type of notFound error for testing.